source/cloud/azure/azure-mnmg.md

```diff
@@ -0,0 +1,70 @@
+# Multi-node, multi-GPU Cluster on Azure
+
+## Create a Cluster using Dask Cloud Provider
+
+The easiest way to setup a multi-node, multi-GPU cluster on Azure is to use [Dask Cloud Provider](https://cloudprovider.dask.org/en/latest/index.html).
+
+### 1. Install Dask Cloud Provider
+
+Dask Cloud Provider can be installed via `conda` or `pip`. The Azure-specific capabilities will need to be installed via the `[azure]` pip extra.
+
+```shell
+$ pip install dask-cloudprovider[azure]
+```
+
+### 2. Configure your Azure Resources
+
+Set up your Azure Resouce Group, Virtual Network, and Security Group according to [Dask Cloud Provider instructions](https://cloudprovider.dask.org/en/latest/azure.html#authentication).
+
+### 3. Create a Cluster
+
+From a Python terminal, a cluster can be created using the `dask_cloudprovider` package. The below example creates a cluster with 2 workers in `westus2` with `Standard_NC12s_v3` VMs. Ensure that the VMs have at least 100GB of disk.
+
+```python
+from dask_cloudprovider.azure import AzureVMCluster
+resource_group = "<RESOURCE_GROUP>"
+vnet = "<VNET>"
+security_group = "<SECURITY_GROUP>"
+subscription_id = "<SUBSCRIPTION_ID>"
+cluster = AzureVMCluster(
+    resource_group=resource_group,
+    vnet=vnet,
+    security_group=security_group,
+    subscription_id=security_group,
+    location="westus2",
+    vm_size="Standard_NC12s_v3",
+    public_ingress=True,
+    disk_size=100,
+    n_workers=2,
+    worker_class="dask_cuda.CUDAWorker",
+    docker_image="rapidsai/rapidsai:cuda11.5-runtime-ubuntu18.04-py3.9",
```

Comment:
This isn't the latest Docker image, I think the Dask Cloudprovider docs are very out of date. The naming scheme changed a few years ago and this tag stopped being updated. I expect this will give `0.19`. 

```suggestion
    docker_image="rapidsai/rapidsai-core:22.12-cuda11.5-runtime-ubuntu20.04-py3.9",
```